app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class MenusController < ApplicationController
+  def update
+    @menu = Menu.find(params[:id])
+    recipe = Recipe.find(params[:recipe_id])
+    @menu.recipes << recipe
+    flash[:success] = "Recipe was added to menu"
+    redirect_to controller: :recipes, action: :index
```

Comment:
if `@menu.recipes << recipe` fails, this will not return an error message. Consider:

```
menu = Menu.find(params[:id])
recipe.menu = menu

if recipe.save
  # success
else
  # error
end

Reply:
I couldn't do it exactly this way but now it renders an error msg if recipe already exists on menu and if something goes wrong